main-settings/src/main/scala/sbt/Scope.scala

```diff
@@ -31,6 +32,9 @@ final case class Scope private (
   def rescope(config: ConfigKey): Scope = copy(config = Select(config))
   def rescope(task: AttributeKey[?]): Scope = copy(task = Select(task))
 
+  final def /[K](key: Scoped.ScopingSetting[K]): K = scope(key)
+  def scope[K](key: Scoped.ScopingSetting[K]): K = key.rescope(this)
+
```

Comment:
Why do we have 2 versions of this method? Should we keep `/` only?

Reply:
For any symbolic operators, I think it would be a good practice to have English method name counterpart that we can explain exactly what they do.